Dockerfile

```diff
@@ -1,12 +1,16 @@
 FROM jupyter/datascience-notebook
 
-MAINTAINER Pierre Bellec <pierre.bellec@gmail.com>
+LABEL maintainer="Pierre Bellec <pierre.bellec@gmail.com>"
 
 USER jovyan
 
-RUN pip install nilearn
+# Copying the repository inside the container
+COPY . /home/jovyan/work
 
-# Get the files of the project
-RUN wget https://github.com/brainhack101/introML/archive/0.1.zip
-RUN unzip 0.1.zip
-RUN rm 0.1.zip
+# Instaling the requirements
+RUN pip install nilearn \
```

Comment:
Is there a reason to split this line ?

Reply:
I think because that was the only python requirement, instead of a full `pip install requirements.txt`.

But this assumed they had created the environment described in the sklearn folder's requirements.txt, so we might want to just make a fresh requirements and install that.

Reply:
Does the notebook run with only nilearn as the dependency ?